rotkehlchen/chain/evm/node_inquirer.py

```diff
@@ -225,7 +225,7 @@ def __init__(
         # A cache for erc20 and erc721 contract info to not requery the info
         self.contract_info_erc20_cache: LRUCacheWithRemove[ChecksumEvmAddress, dict[str, Any]] = LRUCacheWithRemove(maxsize=1024)  # noqa: E501
         self.contract_info_erc721_cache: LRUCacheWithRemove[ChecksumEvmAddress, dict[str, Any]] = LRUCacheWithRemove(maxsize=512)  # noqa: E501
-        self.maybe_connect_to_nodes(when_tracked_accounts=True)
+        self.failed_to_connect_nodes: set[str] = set()
```

Comment:
please add a description either above this or in the docstring of the constructor what this is for and underline the answer to my question from before. Specifically state that if a node is added here only a restart takes it out. No reconnect attempt happens.

And probably we would need a note to handle this as perhaps it makes sense to try again once in an hour or something?

Reply:
https://github.com/orgs/rotki/projects/11?pane=issue&itemId=86405753